pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1679,4 +1679,9 @@ self: super: {
   # Issue reported upstream, no bug tracker url yet.
   darcs = doJailbreak super.darcs;
 
+  github = appendPatch super.github (pkgs.fetchpatch {
```

Comment:
Could you also add a comment above describing why this patch was added and when it can be removed (presumably at the next release if it was picked from github's master branch).

This helps with cleaning up unnecessary overrides.